rke/k8s_rke_system_images.go

```diff
@@ -5651,6 +5651,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled out of band post v2.5.12
```

Comment:
Nit, but v2.5.13 is out so it should be stating that version I think: https://github.com/rancher/rancher/releases/tag/v2.5.13